src/projectFormController.js

```diff
@@ -58,42 +58,45 @@ function createSelect(target, id, options) {
   });
 }
 
-function createProjectInputs() {
+function createInputFields(fields, target) {
+  fields.forEach(field => {
+    let type = 'text';
+    let fieldName;
+
+    if (field.includes('date')){ 
```

Comment:
You could shorten the setting of type to a ternary:
```js
const type = field.includes('date') ? 'date' : 'text'
```

Reply:
both of these solutions you gave are much prettier. making the changes now.